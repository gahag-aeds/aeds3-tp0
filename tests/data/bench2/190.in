190 95
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 29 91 71 83 57 73 47 25 2 90 70 63 76 61 71 79 5 50 37 56 89 46 23 8 16 52 42 30 48 91 12 77 34 84 12 91 9 59 68 12 49 39 27 77 52 50 56 57 0 46 13 41 92 89 1 60 41 43 90 41 86 55 18 20 39 30 63 0 42 32 12 43 23 39 21 75 41 29 84 41 27 50 34 71 91 35 31 32 78 22 
Sub 1 63
Sum 114 176
Sub 91 153
Min 13 75
Add 7 69
Sum 54 116
Sub 52 114
Min 106 168
Add 86 148
Sum 112 174
Add 96 158
Max 15 77
Sub 10 72
Max 37 99
Add 40 102
Min 59 121
Add 23 85
Sum 68 130
Sub 88 150
Min 28 90
Sub 82 144
Max 105 167
Add 116 178
Max 82 144
Add 73 135
Max 124 186
Sub 120 182
Min 87 149
Sub 6 68
Min 68 130
Add 59 121
Min 125 187
Add 54 116
Max 22 84
Sub 2 64
Min 53 115
Sub 67 129
Sum 54 116
Sub 31 93
Max 25 87
Add 30 92
Min 118 180
Add 31 93
Sum 39 101
Add 19 81
Max 63 125
Sub 97 159
Max 41 103
Sub 3 65
Max 50 112
Add 60 122
Max 76 138
Add 19 81
Sum 49 111
Add 30 92
Max 83 145
Add 37 99
Sum 15 77
Add 42 104
Sum 35 97
Add 82 144
Min 92 154
Add 24 86
Sum 127 189
Sub 105 167
Sum 26 88
Sub 44 106
Min 44 106
Sub 71 133
Max 66 128
Add 69 131
Max 100 162
Sub 48 110
Sum 99 161
Sub 4 66
Max 66 128
Sub 34 96
Sum 92 154
Sub 98 160
Max 28 90
Sub 6 68
Min 31 93
Sub 104 166
Max 24 86
Sub 71 133
Min 125 187
Add 107 169
Max 107 169
Sub 12 74
Sum 13 75
Sub 6 68
Sum 85 147
Sub 105 167
Min 22 84
Add 85 147
