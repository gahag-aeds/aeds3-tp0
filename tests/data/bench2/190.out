2824
0
3281
-1
2859
97
98
2
3547
2
92
97
93
-1
1
-1
99
1
3360
97
-1
3096
100
100
100
101
3370
102
2944
3371
-2
2847
3371
5
103
93
3153
103
3179
102
2
101
-5
89
2678
3159
0
